src/bundle/RepositoryInstaller/Command/InstallPlatformCommand.php

```diff
@@ -84,6 +85,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->checkParameters();
         $this->checkCreateDatabase($output);
 
+        $schemaManager = $this->connection->getSchemaManager();
+        if (!empty($schemaManager->listTables())) {
+            $io = new SymfonyStyle($input, $output);
+            if (!$io->confirm('Running this command will delete data in all ibexa generated tables. Continue?', )) {
```

Comment:
```suggestion
            if (!$io->confirm('Running this command will delete data in all Ibexa generated tables. Continue?', )) {
```

This message will be visible to external developers, we use `Ibexa` instead of `ibexa` in our communication